server/api/handlers/fetch_games_in_casino.go

```diff
@@ -3,12 +3,42 @@ package handlers
 import (
 	"context"
 	"encoding/json"
+	"github.com/eoscanada/eos-go"
 	"platform-backend/contracts"
+	"platform-backend/models"
 	"platform-backend/server/api/ws_interface"
+	"strconv"
 )
 
 type FetchGamesInCasinoPayload struct {
-	CasinoId uint64 `json:"casinoId"`
+	CasinoId eos.Uint64 `json:"casinoId"`
+}
+
+type GameParamResponse struct {
+	Type  uint16 `json:"type"`
+	Value string `json:"value"`
+}
+
+type CasinoGameResponse struct {
+	Id     string              `json:"gameId"`
+	Paused bool                `json:"paused"`
+	Params []GameParamResponse `json:"params"`
+}
+
+func toCasinoGameResponse(g *models.CasinoGame) *CasinoGameResponse {
+	ret := &CasinoGameResponse{
+		Id:     strconv.FormatUint(g.Id, 10),
+		Paused: g.Paused,
+	}
+
+	for _, param := range g.Params {
```

Comment:
Here you also already know what size the array will be.
`make(..., len()). for i, param: = range.  ret.Prams[i] =`